### DISCUSSION TITLE
editor/

### DISCUSSION THREAD

**Discussion by giscus** [casbin/casbin#1136] [Docs comments]
# editor/

Casbin Online Editor

https://casbin.org/editor/

**Comment by qq1176914912** [comment]
The online editor is missing.

**Reply by hsluoyz** [reply]
Fixed now

**Comment by szarea** [comment]
why the following request returning false:
 alice, /book/2, GET
bob, /pen/1, GET

https://editor.casbin.org/#5U45BJ42S

**Reply by hsluoyz** [reply]
Hi @szarea this dicussion area is not for discussing about user-specific cases. please create a GitHub issue for your case

**Comment by inesfcnogueira** [comment]
Where should we add the custom casbin function on a node.js project?

**Reply by hsluoyz** [reply]
@inesfcnogueira see the example code: https://github.com/casbin/node-casbin/blob/d086457c090732863c082419a866e8169b4a8cef/test/model.test.ts#L256
